src/test/v2.0/support/open-support-2.0.ts

```diff
@@ -65,6 +74,10 @@ export class OpenControl2_0 implements OpenControl<Context> {
   validateReceivedContext = async (context: ContextSender, expectedContextType: string) => {
     expect(context.context.type).to.eq(expectedContextType, openDocs);
   };
+
+  inValidateReceivedContext = async (context: ContextSender, expectedContextType: string) => {
+    expect(context.context.type).not.to.eq(expectedContextType, openDocs);
+  };
 }
```

Comment:
We should have fdc3.open.ts in the v2.0/advanced directory.  It should contain:

AOpensB4

Reply:
(just check that is true)